templates/taxi/license_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}

```

Comment:
Code Style issue: According to the checklist, it's recommended to use 2 whitespaces indentations in your `.html` files. Please, check your code for correct indentations.